fab/fabfile.py

```diff
@@ -436,6 +436,22 @@ def apply_patch(patchfile=None):
     silent_services_restart(use_current_release=True)
 
 
+@task
+def reverse_patch(patchfile=None):
+    """
+    Used to reverse a git patch created via `git format-patch`. Usage:
+
+        fab <env> reverse_patch:patchfile=/path/to/patch
+
+    Note: Only use this when absolutely necessary.
```

Comment:
Bad things don't actually happen the patch just doesn't get applied and the fab task stops. this should ~never be used, except that for the time being we can't deploy to NIC due to not being able to access the internet, so we're forced to use this method. can update the comment